back/app/models/custom_field.rb

```diff
@@ -242,13 +242,27 @@ def other_option_text_field
     return if options.none?(&:other)
 
     other_field_key = "#{key}_other"
+    title_multiloc = MultilocService.new.i18n_to_multiloc(
+      'custom_fields.ideas.other_text_field.title',
+      locales: CL2_SUPPORTED_LOCALES
+    )
+
+    # Replace {other_option} in the title string with the title of the other option
+    other_option = options.detect { |o| o[:other] == true }
+    replace_string = '{other_option}'
+    replaced_title_multiloc = {}
+    title_multiloc.each do |locale, title|
+      replaced_title_multiloc[locale] = if other_option.title_multiloc[locale.to_s]
+        title.gsub(/#{replace_string}/, other_option.title_multiloc[locale.to_s]) if other_option.title_multiloc[locale.to_s]
+      else
+        title
+      end
+    end
+
```

Comment:
This was a bug that I noticed - previously the text always said "If you picked 'other'..." even if you'd changed the title of the 'other' option